src/server.rs

```diff
@@ -389,6 +397,7 @@ impl<C> SccacheService<C>
 {
     pub fn new(storage: Arc<Storage>,
                handle: Handle,
+               client: &Client,
```

Comment:
looks like a few test calls to this function weren't updated, so tests are failing to compile.